pages/hold/request/[id].tsx

```diff
@@ -225,15 +216,42 @@ export async function getServerSideProps({ params, req, res }) {
       throw new Error("No item id in url")
     }
     const { discoveryBibResult } = await fetchBib(bibId, {}, itemId)
-
     const discoveryItemResult = discoveryBibResult?.items?.[0]
 
     if (!discoveryItemResult) {
       throw new Error("Item not found")
     }
 
+    const bib = new Bib(discoveryBibResult)
+    const item = new Item(discoveryItemResult, bib)
+
+    // Redirect if to aeonUrl if present in the item response
+    if (item.aeonUrl) {
```

Comment:
This is the same as on DFE. we already change the button hrefs, but I'm assuming this is here in case the hold page is accessed directly. Not sure if we still need this.